mail/api_test.py

```diff
@@ -216,3 +240,38 @@ def test_financial_aid_email_with_blank_subject_and_body(self, mock_post):
         assert audit.from_email == settings.MAILGUN_FROM_EMAIL
         assert audit.email_subject == ''
         assert audit.email_body == ''
```

Comment:
Can you test that `generate_financial_aid_email` raises a `ValidationError` for all other statuses?